subprojects/docs/src/docs/userguide/jvm/toolchains.adoc

```diff
@@ -152,6 +152,7 @@ JVM auto-detection knows how to work with:
 * Operation-system specific locations: Linux, macOS, Windows
 * Package Managers: https://asdf-vm.com/#/[Asdf-vm], https://github.com/shyiko/jabba[Jabba], https://sdkman.io/[SDKMAN!]
 * https://maven.apache.org/guides/mini/guide-using-toolchains.html[Maven Toolchain] specifications
+* Installations via https://www.jetbrains.com/help/idea/sdk.html#jdk-from-ide[IntelliJ IDEA]
```

Comment:
Maybe name it "https://www.jetbrains.com/help/idea/sdk.html#jdk-from-ide[IntelliJ IDEA] installations"?

Reply:
Fixed with 0d44e23953a4aba6bc3e6e04e990337a99767778.